hapic/hapic.py

```diff
@@ -74,18 +77,26 @@ def context_getter():
         # who will need error_builder but declared (like with decorator)
         # before error_builder declaration
         def error_builder_getter():
-            return self._context.get_default_error_builder()
+            return self._context.default_error_builder
 
+        self._processor_class = processor_class
         self._context_getter = context_getter
-        self._processor_class = (
-            MarshmallowProcessor
-        )  # type: typing.Type[Processor]  # nopep8
         self._error_builder_getter = error_builder_getter
 
-        # TODO: Permettre la surcharge des classes utilisés ci-dessous, see #14
+    @property
+    def ready(self) -> bool:
+        """
+        :return: True if hapic have a processor_class, else False
+        """
+        return bool(self._processor_class)
 
     @property
     def processor_class(self) -> typing.Type[Processor]:
+        if self._processor_class is None:
+            raise ConfigurationException(
+                "Hapic must be built with a processor_class"
```

Comment:
message must be cleared : "You must define Hapic processor_class before to use it.". The word "built" has no clear meaning in the context of a python module.